src/DTO/ExperimentConfiguration.php

```diff
@@ -135,6 +140,22 @@ public function setOverrides(array $overrides): void
         $this->overrides = $overrides;
     }
 
+    /**
+     * @return array
+     */
```

Comment:
I added these comments to match the rest of the file, however, I think we should axe all of them since they are redundant with the type hints in the code itself.